simulator.py

```diff
@@ -365,13 +365,13 @@ def at(self, location, duration):
         location.humans.add(self)
         self.leaving_time = duration + self.env.now
         self.start_time = self.env.now
-
+        self.area = self.location.area
```

Comment:
Will assigning this `area` to `self` help?

Reply:
Yes, this was a mistake. Area doesn't need to be assigned to self. Fixed this!